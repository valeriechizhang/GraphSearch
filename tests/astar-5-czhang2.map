1   1   1   1   1   1   1   1   1   1   1
90  80  70  60  50  40  30  20  10  5   1
1   1   1   1   1   1   1   1   1   1   1
1   90  80  70  60  50  40  30  20  10  5
1   1   1   1   1   1   1   1   1   1   1
90  80  70  60  50  40  30  20  10  5   1
1   1   1   1   1   1   1   1   1   1   1