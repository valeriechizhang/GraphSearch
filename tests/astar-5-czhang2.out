Solution steps: ['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W']
Solution cost: 46
Number of states considered: 49

Frontier:
Pos=(0, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'N'] Cost=7946
Pos=(1, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=7943
Pos=(0, 1) Moves=['N'] Cost=7922
Pos=(1, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'N'] Cost=6267
Pos=(2, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=6262
Pos=(1, 1) Moves=['E', 'N'] Cost=6243
Pos=(2, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'N'] Cost=4788
Pos=(3, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=4781
Pos=(2, 1) Moves=['E', 'E', 'N'] Cost=4764
Pos=(3, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'N'] Cost=3509
Pos=(4, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=3500
Pos=(3, 1) Moves=['E', 'E', 'E', 'N'] Cost=3485
Pos=(4, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'N'] Cost=2430
Pos=(5, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=2419
Pos=(4, 1) Moves=['E', 'E', 'E', 'E', 'N'] Cost=2406
Pos=(5, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=1551
Pos=(6, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'N'] Cost=1538
Pos=(5, 1) Moves=['E', 'E', 'E', 'E', 'E', 'N'] Cost=1527
Pos=(6, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=872
Pos=(7, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'N'] Cost=857
Pos=(6, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=848
Pos=(7, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=393
Pos=(8, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'N'] Cost=376
Pos=(7, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=369
Pos=(8, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=114
Pos=(9, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'W'] Cost=55
Pos=(8, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'W'] Cost=52
Pos=(9, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=50

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(1, 0) Moves=['E'] Cost=1
Pos=(2, 0) Moves=['E', 'E'] Cost=2
Pos=(3, 0) Moves=['E', 'E', 'E'] Cost=3
Pos=(4, 0) Moves=['E', 'E', 'E', 'E'] Cost=4
Pos=(5, 0) Moves=['E', 'E', 'E', 'E', 'E'] Cost=5
Pos=(6, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E'] Cost=6
Pos=(7, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=7
Pos=(8, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=8
Pos=(9, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=9
Pos=(10, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=10
Pos=(10, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=11
Pos=(10, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N'] Cost=12
Pos=(9, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W'] Cost=13
Pos=(8, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W'] Cost=14
Pos=(7, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W'] Cost=15
Pos=(6, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W'] Cost=16
Pos=(5, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W'] Cost=17
Pos=(4, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=18
Pos=(3, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=19
Pos=(2, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=20
Pos=(1, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=21
Pos=(0, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=22
Pos=(0, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N'] Cost=23
Pos=(0, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N'] Cost=24
Pos=(1, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E'] Cost=25
Pos=(2, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E'] Cost=26
Pos=(3, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E'] Cost=27
Pos=(4, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E'] Cost=28
Pos=(5, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E'] Cost=29
Pos=(6, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=30
Pos=(7, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=31
Pos=(8, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=32
Pos=(9, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=26
Pos=(9, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=33
Pos=(10, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N'] Cost=29
Pos=(10, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=34
Pos=(10, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=35
Pos=(10, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N'] Cost=36
Pos=(9, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W'] Cost=37
Pos=(8, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W'] Cost=38
Pos=(7, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W'] Cost=39
Pos=(6, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W'] Cost=40
Pos=(5, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W'] Cost=41
Pos=(4, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=42
Pos=(3, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=43
Pos=(2, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=44
Pos=(1, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=45
Pos=(0, 6) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W'] Cost=46
